toolkits/global/packages/global-buttons/scss/10-settings/buttons.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ *  Button settings
+ *
+ */
+
+$button-base--text-align: left;
+$button-base--padding: 8px;
+$button-base--font-size-normal: 16px;
+$button-base--line-height: 1.4;
+$button-base--font-family: sans-serif;
+$button-base--border-radius: 2px;
+$button-shared--border-width: 1px;
+$button-shared--border-style: solid;
+$button-shared--transition: 0.25s ease, color 0.25s ease, border-color 0.25s ease;
+$button--font-size-small: 14px;
```

Comment:
rem?

Reply:
Personal opinion, I think it reads easier as `$component--modifier-property`.
`$button--small-font-size`